env/AWSPW.env

```diff
@@ -16,7 +16,7 @@ step=$1
 
 export npe_node_max=36
 export launcher="mpiexec.hydra"
-export mpmd_opt=""
+export mpmd_opt="--multi-prog --output=mpmd.%j.%t.out"
```

Comment:
Are these the right options for `mpiexec.hydra`?  They appear to be slurm options

Reply:
@rajdpanda Can you please address @aerorahul's questions?

Reply:
that's correct! they are options for srun but not for mpiexec.hydra. Forgot to mention this to Henry. It can be taken out

Reply:
@rajdpanda So, the only change required is to `ush/fv3gfs_downstream_nems.sh`?